builders/pallets-precompiles/precompiles/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Solidity Precompiles
+description: An overview of the available Solidity precompiles on Moonbeam. Precompiles enable you to interact with Substrate features using the Ethereum API.
+---
+
+# Overview of the Precompiled Contracts on Moonbeam
+
+![Precompiled Contracts Banner](/images/builders/pallets-precompiles/precompiles/overview/overview-banner.png)
+
+## Overview {: #introduction } 
+
+On Moonbeam, a precompiled contract is native Substrate code that has an Ethereum-style address and can be called using the Ethereum API, like any other smart contract. The precompiles provide access to Substrate pallets that are coded in Rust and not normally accessible from the Ethereum side of Moonbeam.
```

Comment:
Actually I was thinking about this and precompiles allows you to "call the runtime directly" . For example, the batch precompile does not call any pallet, but some rust logic